docs/content/installation/install_compatibility_requirements.md

```diff
@@ -2,7 +2,7 @@
 
 For the complete and up-to-date information about the compatibility and requirements for using the IBM® block storage CSI driver, refer to its latest release notes. The release notes detail supported operating system and container platform versions, and microcode versions of the supported storage systems.
 
-Before beginning the installation of the CSI (Container Storage Interface) driver, be sure to verify that you comply with the following prerequisites.
+Be sure to verify that you comply with all of the following prerequisites before beginning the installation of the CSI (Container Storage Interface) driver.
```

Comment:
could `all` be interpreted as "including the optional ones"?

Reply:
no this is OK